lib/graph.ml

```diff
@@ -75,17 +75,14 @@ let get_neighbors_toroidal_grid t w l x y a =
     (n4, t, a.(snd n4).(fst n4));
   ]
 
-let init_toroidal_grid t w l a =
+let init_toroidal_grid t l w a =
```

Comment:
I preferred the old version row and column to the new version column and row. But as you want.

Reply:
you're right I'll change that again